OneCommunityGlobal/HighestGoodNetworkApp
# Yongjian create dark mode for app

# Description
![Screen Shot 2023-05-19 at 2 30 58 PM](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46613683/97d89423-fce5-43af-bdff-6f34e32350af)


## Mainly changes explained:
Created a toggle button to change the entire app between light and dark mode and added CSS style changes for dark mode to designated components.

## How to test:
1. check into current branch
2. do `npm run start:local` to run this PR locally
3. login as admin role and toggle to dark mode and inspect the entire application for changes.

## Demo
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46613683/5d6c2db2-66c6-4376-b17f-ac6f0e9e1b82

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46613683/d18ad931-80e7-45a0-9ed8-d30eee880cb6
